tools/wptserve/wptserve/server.py

```diff
@@ -285,6 +285,9 @@ def finish_handling(self, request, response, handler):
             try:
                 handler(request, response)
             except HTTPException as e:
+                if e.code // 100 == 5:
```

Comment:
This seems unnecessarily clever compared with e.g. `if 500 <= e.code < 600`

Reply:
Agreed! Fixed in next commit.